lib/ometajs/api.js

```diff
@@ -117,12 +124,14 @@ function evalCode(code, filename, options) {
 };
 exports.evalCode = evalCode;
 
-// Allow users to `require(...)` ometa files
-require.extensions['.ometajs'] = require.extensions['.ojs'] = require.extensions['.ometa'] = function(module, filename) {
-	var code = compile(require('fs').readFileSync(filename).toString(), {
-		nodeRequirePath: __dirname + '/core',
-		nodeRequireProperty: ''
-	});
+if (require && require.extensions) {
+	// Allow users to `require(...)` ometa files
+	require.extensions['.ometajs'] = require.extensions['.ojs'] = require.extensions['.ometa'] = function(module, filename) {
+		var code = compile(require('fs').readFileSync(filename).toString(), {
```

Comment:
This should only be called in a commonjs environment so I think it would make sense to add the commonjs flag here

Reply:
What if the user doesn't set the `commonjs` flag, but loads `api.js`? I found this situation in some of my cases. 

The above code seems like it will be more robust; it guarantees you won't trigger a ReferenceError in any situation.